src/Components/Aspire.Azure.Storage.Blobs/ConfigurationSchema.json

```diff
@@ -27,75 +27,156 @@
+                            },
+                            "LoggedContentSizeLimit": {
+                              "type": "integer",
+                              "description": "Gets or sets value indicating maximum size of content to log in bytes. Defaults to 4096."
+                            }
+                          },
+                          "description": "Gets the client diagnostic options."
+                        },
                         "EnableTenantDiscovery": {
                           "type": "boolean",
-                          "description": "Enables tenant discovery through the authorization challenge when the client is configured to use a TokenCredential. When enabled, the client will attempt an initial un-authorized request to prompt a challenge in order to discover the correct tenant for the resource."
+                          "description": "Enables tenant discovery through the authorization challenge when the client is configured to use a TokenCredential.\r\n            When enabled, the client will attempt an initial un-authorized request to prompt a challenge in order to discover the correct tenant for the resource."
                         },
                         "EncryptionScope": {
                           "type": "string",
-                          "description": "Gets the EncryptionScope to be used when making requests."
+                          "description": "Gets the P:Azure.Storage.Blobs.BlobClientOptions.EncryptionScope to be used when making requests."
                         },
                         "GeoRedundantSecondaryUri": {
                           "type": "string",
                           "format": "uri",
-                          "description": "Gets or sets the secondary storage Uri that can be read from for the storage account if the account is enabled for RA-GRS."
+                          "description": "Gets or sets the secondary storage T:System.Uri that can be read from for the storage account if the\r\n             account is enabled for RA-GRS.\r\n            \r\n             If this property is set, the secondary Uri will be used for GET or HEAD requests during retries.\r\n             If the status of the response from the secondary Uri is a 404, then subsequent retries for\r\n             the request will not use the secondary Uri again, as this indicates that the resource\r\n             may not have propagated there yet. Otherwise, subsequent retries will alternate back and forth\r\n             between primary and secondary Uri."
```

Comment:
Interesting, I'm curious, how does this render in the IDE, I suppose it actually shows the new lines as opposed to having the new line characters printed out?

Reply:
It renders the newlines correctly. This is an area that will probably need to be tweaked over time. I don't know why the newlines and extra spaces are in there. I need to look at some more Roslyn code to figure out how they strip out all the whitespace in these XML docs.

![image](https://github.com/dotnet/aspire/assets/8291187/da8b094d-7a8e-4286-9730-008ebdaff7f8)